bin/isolated-functions.sh

```diff
@@ -23,7 +23,11 @@ fi
 shopt -s expand_aliases
 
 assert() {
-	IFS='|' expression=${PIPESTATUS[*]} let '! expression' || die "$@"
+	local x pipestatus=( "${PIPESTATUS[@]}" )
+	___eapi_has_assert || die "'${FUNCNAME}' banned in EAPI ${EAPI}"
```

Comment:
Shouldn't we also adjust usages of `assert` by portage code when introducing this? Like I did in the alternative PR to this https://github.com/gentoo/portage/pull/1453, for example https://github.com/gentoo/portage/pull/1426/files#diff-4eacf848f4d2cbea61f0c0e5285e082233b162cd5d096566a20947f484961b4dL543-R543